### PR TITLE
Some new props

### PR DESCRIPTION
**Pull request by vulkans22**
Here's the first bunch of conversions, originally meant to cover the OPT-J HT IVA I've been working on (Included in FlightSystems/MAS_JHT.cfg) but since I was already close to it, also covers Alex's ASET MK1 cockpit internal. I've left a renamed copy of the MM replacer patch I was using (MAS_ASET/ASET_MAS_MM.nocfg) for quick testing. Also left loading of any LUA scripts in a separate file (MAS_ASET/LUA_Scripts.cfg) so as to not contaminate ASET_Props.cfg.

Issues present are the air flow intake gauge (ASET_IntakeAir) which I could not figure out a way to replicate exactly, so for now it's just a fancy IntakeAir percentage radial gauge.

Let me know of any necessary changes.

### CODE REVIEW

**Comment by MOARdV** [comment]
That's a lot of files... It might take me a couple of days to review and merge. :)

I will get some feedback to you, probably closer to the end of the week.  I've got a master Word document file that's intended to be an IVA maker's guide to the props (it's local, not on GitHub), and I need to make sure I document all of the new props so future IVA makers don't need to spend as much time guessing which props they want to use.

**Comment by MOARdV** [comment]
(adding comment to link this PR to Issue #277 )

**Comment by vulkans22** [comment]
Take your time, I'm free for the foreseeable future to lend a hand in any way.

With all the different variations of existing props, a master document for everything sounds like a great idea for sure, I end up spending half my time on just searching for specific items in parttools. I'm thinking some type of baseline for filename and folder organization would also be desirable.

**Comment by MOARdV** [comment]
@vulkans22 - First, let me apologize profusely for letting this change sit for five months.  I had a lot going on IRL, and I was a bit burned out on dealing with prop config files.

I'm going over this pull request now.  So far, the only issue I've seen were a few files where the prop's name did not have the MAS_ prefix.  Considering how many files there are, that's not a big deal.

If you're available to address renaming those prop names, I'll mark the files for you.  Otherwise, I'll commit the changes and edit them afterwards.

File: GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-COCKPIT.cfg [hunk 1]
```diff
@@ -0,0 +1,124 @@
+PROP
+{
+  name = ASET_B-Knob_LD_COCKPIT
```
Comment (MOARdV) [inline GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-COCKPIT.cfg:3]: Need MAS_ prefix

File: GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-CP_CMD.cfg [hunk 2]
```diff
@@ -0,0 +1,124 @@
+PROP
+{
+  name = ASET_B-Knob_LD_CP-CMD
```
Comment (MOARdV) [inline GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-CP_CMD.cfg:3]: Needs MAS_ prefix

File: GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-CP_LEFT.cfg [hunk 3]
```diff
@@ -0,0 +1,124 @@
+PROP
+{
+  name = ASET_B-Knob_LD_CP-LEFT
```
Comment (MOARdV) [inline GameData/MOARdV/MAS_ASET/ASET_B-Knob/ASET_B-Knob-LightDimmer/ASET_B-Knob_LD-CP_LEFT.cfg:3]: Needs MAS_ prefix

File: GameData/MOARdV/MAS_ASET/Switch_Toggle_Modular/tggl_ABORT_T1-G3-B15-C1Y-S/tggl_ABORT_T1-G3-B15-C1Y-S.cfg [hunk 4]
```diff
@@ -0,0 +1,201 @@
+PROP
```
Comment (MOARdV) [inline GameData/MOARdV/MAS_ASET/Switch_Toggle_Modular/tggl_ABORT_T1-G3-B15-C1Y-S/tggl_ABORT_T1-G3-B15-C1Y-S.cfg:1]: I think this config file got moved into a subdirectory with the same name.

**Pull request merged**